### PR TITLE
Adding my project for MGY441 (Obeliscus porcinus)

### PR DESCRIPTION
**Pull request by dzheglov**
Cluster 36

### CODE REVIEW

File: virus/obeliscus_porcinus.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+In our analysis, we classified the genomes of the first 100 obelisk clusters into distinct groups based on their coding potential and genomic architecture. While many obelisks encode only one or two proteins, some, like cluster 36, may encode a third protein on the antisense strand. Others, such as cluster 33, exhibit evidence of recent genome duplication, producing a symmetrical genome that encodes identical proteins on the sense and antisense strands.  These results indicate that there is a high degree of variability in the evolution of the obelisk family and may suggest the potential for new ways of protein coding in these RNA elements.
+
+The ecological distribution of omega36 also strengthens the percription of omega36 as an obelisk RNA. Thus, omega36 was detected in geographically different datasets originated from China and the USA and was linked with porcine microbiome and bacterial hosts.
+
+## Results
+
+### Ecology subsection A...
```
Comment (ababaian) [inline virus/obeliscus_porcinus.md]: Amend out the "Default Titles" from the document. Your descriptive titles are much better throughout.

File: virus/obeliscus_porcinus.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+![Figure 3. Open Reading Frames through ORFfinder.](img/O_porcinus_omega36/ORFfinder.png)
+Figure 3. Open Reading Frames through ORFfinder.
+
+
+
+For annotation purposes, I focused on ORF8 on the reverse strand, which encodes a potential reverse protein, and ORF3, a small peptide with promising AlphaFold structural predictions. These features were annotated using SnapGene for a clear visualization of their genomic positions and orientations.
+
+Here are the protein sequences:
+
+Oblin-1: MRINQTVDEFNLDLYRNRDDWETIQLLDGTVYEKVSWYTEDGKYQEYIHK
```
Comment (ababaian) [inline virus/obeliscus_porcinus.md]: For sequences, please use `code text` or for longer sequences
```
code block which gives a nice fixed-width font aesthetic.
```

**Review by ababaian — CHANGES_REQUESTED** [review]
Looks like a great submission! Can't wait to deep dive into it. A few minor fixes to close the PR

File: virus/obeliscus_porcinus.md (hunk 2 above)
Comment (dzheglov) [reply virus/obeliscus_porcinus.md]: Hello! Thank you for the feedback! I just added the requested changes. I hope that it's not too late :)

**Review by ababaian — APPROVED** [review]

**Pull request merged**